bundles/org.openhab.binding.energidataservice/README.md

```diff
@@ -380,9 +347,26 @@ Number GridTariff "Current Grid Tariff" <price> (TotalPrice) { channel="energida
 Number SystemTariff "Current System Tariff" <price> (TotalPrice) { channel="energidataservice:service:energidataservice:electricity#system-tariff" [profile="transform:VAT"] }
 Number TransmissionGridTariff "Current Transmission Grid Tariff" <price> (TotalPrice) { channel="energidataservice:service:energidataservice:electricity#transmission-grid-tariff" [profile="transform:VAT"] }
 Number ElectricityTax "Current Electricity Tax" <price> (TotalPrice) { channel="energidataservice:service:energidataservice:electricity#electricity-tax" [profile="transform:VAT"] }
-String HourlyPrices "Hourly Prices" <price> { channel="energidataservice:service:energidataservice:electricity#hourly-prices" }
 ```
 
+### Persistence Configuration
+
+```java
+Strategies {
+    default = everyChange
+}
+
+Items {
+    SpotPrice,
+    GridTariff,
+    SystemTariff,
+    TransmissionGridTariff,
+    ElectricityTax: strategy = forecast
+}
+```
+
+In case persistence is only needed for charts and/or accessing prices from rules, [InMemory Persistence](https://www.openhab.org/addons/persistence/inmemory/) can be used.
```

Comment:
Many thank for providing this example, this is something missing in " InMemory Persistence" service.